### ISSUE TITLE
Analysis: Median split on PSS

### ISSUE THREAD

**Issue by psokolhessner** [sokolhessnerlab/bst#1]
Some exploratory PSS analyses suggested nonlinearities in the effects of chronic stress (measured with the Perceived Stress Scale [PSS]) on trust perception & decisions. Current standard categorical delineations of PSS scores into low, moderate, and high severity don't work well because only one participant falls into the "high" severity group. 

**Describe the solution you'd like**
Perform a median split analysis on PSS. 

- [x] Delineate participants into two groups: those above or at the median PSS value, and those below, and store this binary variable (`pssMedianSplit`; low -1, high +1).
- [x] Make histogram of PSS scores with median value vertical line (like that for current categorical distinctions)
- [x] Calculate basic analyses of high and low groups (mean, median, sd.)
- [x] Use median split PSS in new versions of existing trust rating & trust game & sequential-effects analyses. 

**Describe alternatives you've considered**
A tripartite split (into 3 equal [or as equal as possible] groups). Analysis is harder, as this approach still implicitly assumes linearity across the 3 levels.

**Comment by EvrimBaykal** [comment]
@psokolhessner The median split into a binary variable ("low" -1, "high" +1) has been implemented and stored as a new variable.  Histograms and basic analyses have been completed.  It looks like PSS as a continuous measure was utilized for the trust rating & trust game analyses, not the median split method as suggested above.  Should we create a new issue for that step or continue with the updates to chronic stress/trust analyses via this issue?

**Comment by psokolhessner** [comment]
@EvrimBaykal Given that we are making more (bigger?) changes to the dataset (i.e. chasing down partial data, etc), I'd simply slot this under "part of a new approach to trust analysis" that we'll do when we approach that section of the analysis (where we integrate stress & trust data). Does that make sense? So much needs to happen there, that opening new issues for each analysis we do will be too much. 

TL;DR: this is great, close the issue, no need to make new issue. _Using_ this new variable will entail many changes in many analyses down the road - this is just about creating it & prepping it. 

I'll let you close the issue!

**Comment by EvrimBaykal** [comment]
@psokolhessner Sounds good!
